### DISCUSSION TITLE
Default `as` prop for variants

### DISCUSSION THREAD

**Discussion by benknab** [chakra-ui/chakra-ui#5478] [Q&A]
I tried to add an `as` prop for my heading variants but it didn't work:
```ts
...
heading1: {
      as: 'h1',
      fontWeight: 'medium',
      fontSize: '48px',
      lineHeight: '56px',
      letterSpacing: '-0.04em',
    },
...
```
Is it possible to define a default `as` prop for variants?

**Comment by TomasSestak** [comment]
Wish this would be answered - seems like basic feature
